### DISCUSSION TITLE
qterminal text does not go fully all the way to the right side

### DISCUSSION THREAD

**Discussion by Raj2032** [lxqt/lxqt#2312] [Q&A]
Running xfce over here,

![image](https://user-images.githubusercontent.com/29293830/192097235-4e85120b-2df1-4e4c-a411-efc35c0a9807.png)

See how there is this random empty gap with qterminal when running htop?

I essentially copied over `qtermwidget5` from Kali to Peppermint (the current distro I am using), `qterminal.ini` and `qt5ct.conf`.

The theme is working fine but yeah I have this random gap now so just wondering if anyone had any ideas?

**Comment by Raj2032** [comment]
Ok so an update,

![image](https://user-images.githubusercontent.com/29293830/192103844-a4f450ae-cf48-4885-a3dd-feb11b018e42.png)

Fira code is not showing up on Peppermint like on Kali:

![image](https://user-images.githubusercontent.com/29293830/192103870-bfb9980d-5e96-4797-aabe-52c6a0cf56e0.png)

I installed it using this method:

[https://github.com/tonsky/FiraCode/wiki/Linux-instructions#installing-with-a-package-manager](https://github.com/tonsky/FiraCode/wiki/Linux-instructions#installing-with-a-package-manager)

I just simply typed `sudo apt install fonts-firacod` and it installed but yeah its not showing as an option in `qterminal` but yeah not seeing the options, any ideas?

**Reply by yan12125** [reply]
Those unusal spaces look like a result of non-monospace font.

> Fira code is not showing up on Peppermint like on Kali

I'm not sure how fonts are handled on Kali Linux. On Arch Linux, https://archlinux.org/packages/community/any/ttf-fira-code/ is correctly detected as a monospace font and appears in qterminal's dialog.

As a side note, qterminal relies on Qt for filtering monospace fonts: https://github.com/lxqt/qterminal/blob/8df870c42b7b9229876b6b5ecdc72ac30599ed7a/src/fontdialog.cpp#L26

**Reply by Soundtoxin** [reply]
You may need to run `fc-cache -rfv` after installing a new font before other programs can see it.
